au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/audio/BalanceTooltip.qml

```diff
@@ -39,22 +44,27 @@ StyledPopupView {
 
     FontMetrics {
         id: fontMetrics
+
         font: content.font
```

Comment:
` qrc:/qml/Audacity/ProjectScene/trackspanel/audio/BalanceTooltip.qml:42:9: Unable to assign [undefined] to QFont`

Reply:
Thanks, fixed!